pkg/utils/taints.go

```diff
@@ -51,15 +51,22 @@ func CreateFARNoExecuteTaint() corev1.Taint {
 	}
 }
 
+// CreateOutOfServiceTaint returns an OutOfService taint
+func CreateOutOfServiceTaint() corev1.Taint {
+	return corev1.Taint{
+		Key:    v1alpha1.OutOfServiceTaintKey,
+		Effect: corev1.TaintEffectNoExecute,
```

Comment:
IIUC the taint also has a value (`nodeshutdown`) that we should include in the taint. This addition won't affect the taint effect, since the taint is unique by Key and Effect, but I believe it makes sense to follow standards as using this value https://kubernetes.io/blog/2023/08/16/kubernetes-1-28-non-graceful-node-shutdown-ga/#what-is-a-non-graceful-node-shutdown

>kubectl taint nodes <node-name> node.kubernetes.io/out-of-service=nodeshutdown:NoExecute

Reply:
It was addressed above :)